tests/vuefire.spec.js

```diff
@@ -25,6 +27,36 @@ describe('VueFire', function () {
     })
   })
 
+  describe('is callable as Function', function () {
```

Comment:
The title is missing something about __option__, which is what you're actually changing. I'd name it something like _support Function options_